source/administration/config-settings.rst

```diff
@@ -4714,6 +4714,21 @@ Audit settings
 
 The audit settings output audit records to syslog (local or remote server via TLS) and/or to a local file. Both are disabled by default. They can be enabled simultaneously.
 
+Remote Clusters
```

Comment:
@cwarnermm Just confirming this is under the experimental section - I believe so but may have missed the heading in my scrolling.

Reply:
@wiersgallak Confirmed. This new config setting is documented under the "Experimental Settings only in config.json" section.